src/jira.py

```diff
@@ -175,11 +153,13 @@ def update_remote(self):
         if r.status_code != 204:  # HTTP 204 No Content on success
             print(f'{r.status_code} Error transitioning')
 
-        # Issue assignee, description, summary, and story points fields can be updated from a dictionary
-        r = requests.put(f'{self.repo.url}issue/{self.jira_key}', headers=self.repo.headers, json=self.dict_format())
+        # Story points field can be updated from a dictionary
+        if self.story_points:  # Do not try to update this if there is no value for story points
+            r = requests.put(f'{self.repo.url}issue/{self.jira_key}', headers=self.repo.headers,
```

Comment:
Not sure why we don't use function `api_call` from the `Repo` class? Also, I would change `r` to `response`.